_pages/tutorial.markdown

```diff
@@ -16,37 +16,193 @@ toc: true
 
-## Tutorial on doing this and that
-This is much easier than that.
+By pressing enter you get search details. Results contain label, type (term, vocabulary, document...) and source. Clicking a term shows its details.
+
+#### Browsing vocabularies
+
+Another way is browsing vocabularies. Click on "vocabularies" in the left panel to see the list of vocabularies. Above the list is a text field for filtering. Use it to filter out the vocabulary called 'Vocabulary of Prague Building Regulations 2016'.
+
+{% include figure image_path="/assets/images/tutorial/vocabularies.png" alt="Filtering vocabulary" caption="Filtering vocabulary." %}
+
+Clicking the vocabulary shows its detail. Below the basic details is open tab 'Terms' with the list of all terms in a hierarchic structure. Empty text box above allows filtering terms.
+
+{% include figure image_path="/assets/images/tutorial/terms-in-vocab.png" alt="Browsing terms in a vocabulary" caption="Browsing terms in a vocabulary." %}
+
+Click the term to show its detail.
+
+#### Term detail
+
+Term detail contains basic information about the term, hierarchic structure of other terms in the same vocabulary and detailed information about the term in tabs in the lower part of the page.
+
+{% include figure image_path="/assets/images/tutorial/detail-1.png" alt="Term details" caption="Term details." %}
+
+### Marking occurrences of terms and their definitions in documents
+
+Vocabularies are usually interconnected to the documents containing their definitions and occurrences. Open  'Vocabularies' in the left panel and search for the vocabulary 'Vocabulary of Building Act no. 72/2004 of the New Zealand'. In the vocabulary detail open tab 'Document'.
+
+{% include figure image_path="/assets/images/tutorial/vocab-detail.png" alt="Detail of a document assigned to the vocabulary" caption="Detail of a document assigned to the vocabulary." %}
+
+Document information contain document label, its description and list of assigned terms. Those terms have marked occurrences in the document. Below is a list of files belonging to the document. Building Act has only one called 'building-act.html'. By clicking 'Show Content' next to the label you get to the file content.
+
+{% include figure image_path="/assets/images/tutorial/document-content.png" alt="Content of a document with marked term occurrences and definition" caption="Content of a document with marked term occurrences and definition." %}
+
+Green colored terms are occurrences of terms, blue colored are definitions of terms. Click on them to show pop-up with information which term is marked. You can remove or edit the occurrence in the pop-up. Clicking the label redirects you to the term detail.
+
+{% include figure image_path="/assets/images/tutorial/occurrence.png" alt="Occurrence detail" caption="Occurrence detail." %}
+
+[comment]: <> TODO: přidat označování výskytů a definic
```

Comment:
TODO ?

Reply:
that's a comment for future enhancing (it's not visible in the page)